python/src/writeYourProgram.py

```diff
@@ -33,6 +33,22 @@ def _debug(s):
 Callable = typing.Callable
 dataclass = dataclasses.dataclass
 
+# Reexports for Untypy
+try:
+    from untypy import unchecked, precondition, postcondition
```

Comment:
At other places, the ImportError is not caught.

Reply:
@CodeSteak is there a reason for catching the ImportError here but not elsewhere?

Reply:
This was to support importing wyyp without untypy. For the runner script this seemed unneeded, since untypy is "bundled" with it.